apps/api/README.md

```diff
@@ -0,0 +1,82 @@
+# ZKchainHub API
+
+## Overview
+
+The `@zkchainhub/api` app is an Express server that exposes an API where you can fetch information about ZKsync ecosystem and their chains.
+
+## 📋 Prerequisites
+
+-   Ensure you have `node >= 20.0.0` and `pnpm >= 9.5.0` installed.
+
+## Installation
+
+```bash
+$ pnpm install
+```
+
+## Building
+
+To build the monorepo packages, run:
+
+```bash
+$ pnpm build
+```
+
+## ⚙️ Setting up env variables
+
+-   Create `.env` file and copy paste `.env.example` content in there.
+
+```
+$ cp .env.example .env
+```
+
+Available options:
+
+-   (optional) `PORT` on which API is made available. By default is port 3000
+-   (optional) `ENVIRONMENT` which environment we are using ( 'mainnet' | 'testnet' | 'local'). By default is 'mainnet'
+-   `BRIDGE_HUB_ADDRESS`
+-   `SHARED_BRIDGE_ADDRESS`
+-   `STATE_MANAGER_ADDRESSES` CSV list of State managers addresses
+-   `L1_RPC_URLS` as CSV list of RPC URLs. For example, `https://eth.llamarpc.com,https://rpc.flashbots.net/fast`. You can check [Chainlist](https://chainlist.org/) for a list of public RPCs
+-   (optional) `PRICING_SOURCE` which pricing source to use ('dummy' | 'coingecko'). By default is dummy
+-   (optional) `DUMMY_PRICE` for dummy pricing source. Default is undefined
+-   (required if 'coingecko' is selected)`COINGECKO_API_KEY`, `COINGECKO_BASE_URL` and `COINGECKO_API_KEY` depending on your API plan. You can get an API Key creating an account on [Coingecko's site](https://www.coingecko.com/en/api)
+-   `METADATA_SOURCE` which metadata source to use ('github' | 'local' | 'static')
+-   (required if METADATA_SOURCE is 'github') `METADATA_TOKEN_URL` Metadata tokens URL
+-   (required if METADATA_SOURCE is 'github') `METADATA_CHAIN_URL` Metadata chains URL
+-   (required if METADATA_SOURCE is 'local') `METADATA_TOKEN_JSON_PATH` Metadata tokens JSON file path (see examples on `packages/metadata`)
+-   (required if METADATA_SOURCE is 'local') `METADATA_CHAIN_JSON_PATH` Metadata chain JSON file path (see examples on `packages/metadata`)
```

Comment:
This is going the **extra extra mile** but, what do you think about using a Markdown table for this part? You can use [this tool](https://www.tablesgenerator.com/markdown_tables) to generate your tables in an easier way

Eg
| Name               | Description                                          | Default   | Notes |
|--------------------|------------------------------------------------------|-----------|-------|
| PORT               | Port on which API is made available.                 | "3000"    |       |
| ENVIRONMENT        | Environment used ('mainnet' \| 'testnet' \| 'local') | "mainnet" |       |
| BRIDGE_HUB_ADDRESS |                                                      |           |       |
| ...                |                                                      |           |       |

Reply:
thanks 🫶